### PR TITLE
Add Real Time Notifications section in Realtime introduction

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: docs/Realtime.md [hunk 1]
```diff
@@ -97,6 +97,96 @@ And the following components:
...
+                Recompute
+            </Button>
+            {progress > 0 && (
+                <Card sx={{ m: 2, maxWidth: 400 }}>
+                    <Alert severity={progress === 100 ? 'success' : 'info'}>
+                        <AlertTitle>
+                            Recomputing stats{' '}
+                            {progress === 100 ? 'complete' : 'in progress'}
+                        </AlertTitle>
+                        <LinearProgressWithLabel value={progress} />
```
Comment (slax57) [inline docs/Realtime.md:180]: You did not include the code for `<LinearProgressWithLabel>`

**Review by slax57 — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Review by slax57 — APPROVED** [review]

File: docs/Realtime.md [hunk 2]
```diff
@@ -97,6 +97,112 @@ And the following components:
...
+        }).then(({ json }) => ({ data: json }));
+    }
+}
+```
+
+Then, make sure your API sends events with a topic named `recompute_PROJECT_ID` where `PROJECT_ID` is the project identifier:
+
+```json
+{
+    "type": "recompute_PROJECT_ID",
```
Comment (fzaninotto) [inline docs/Realtime.md:128]: the sentence above mentions "topic", but the snippet uses "type". Is it normal?

File: docs/Realtime.md [hunk 3]
```diff
@@ -97,6 +97,112 @@ And the following components:
...
+    "payload": {
+        "progress": 10
+    },
+}
+```
+
+Finally, create a component to actually call this function and show a notification, leveraging the [useSubscribeCallback](./useSubscribeCallback.md) hook:
```
Comment (fzaninotto) [inline docs/Realtime.md:135]: > to actually call this function

Nos sure which function you're talking about

**Pull request merged**